hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/shell/TestBulkDeleteCommand.java

```diff
@@ -64,4 +78,52 @@ public void testArguments() throws IOException, URISyntaxException {
         Assertions.assertThat(bulkDeleteCommand.childArgs.get(0)).
                 describedAs("Children arguments must match").isEqualTo(arg2);
     }
+
+    @Test
+    public void testLocalFileDeletion() throws IOException {
+        String deletionDir = "toDelete";
+        String baseFileName = "file_";
+        Path baseDir = new Path(testRootDir, deletionDir);
+        List<String> listOfPaths = new ArrayList<>();
+        for(int i = 0; i < 100; i++) {
+            Path p = new Path(baseDir, baseFileName + i);
+            lfs.create(p);
+            listOfPaths.add(p.toUri().toString());
```

Comment:
maybe use a list to verify first if the files are created properly and then use the same list to delete. 
same for below test.

Reply:
Done, coursing through the directory and then deleting the files.

Reply:
What I meant is assert first that you files are created successfully like Assertions.assertThat(filesCreated).isEqualTo(originalFiles).hasCount(x)

Your delete test won't even run if creation fails silently in the first few lines.